.github/workflows/contracts-deploy.yml

```diff
@@ -4,25 +4,55 @@ name: contracts-e2e-tests-and-deploy
 on:
```

Comment:
Do we have some link on GH that this run was tested?

Reply:
https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/4470947462/jobs/7855182573